docs/usage/filters/interactive-output-filtering.md

```diff
@@ -0,0 +1,32 @@
+---
+id: interactive-output-filtering
+title: "Interactive output filtering"
+sidebar_position: 3
+---
+
+Sometimes you can get long response from a certain command and you may want to filter that to see a subset of the actual result.
+while you can achieve that with `--filter` param at the end of your command, there is another option to handle that with interactive
+output filtering. Interactivity is achieved via an input action text box where you can add your filter criteria as text and press `Enter`.
+
+:::info
+Output Filter input text box will be attached to your command response if response has more than 15 lines. Let's see an example for this situation.
+:::
+
```

Comment:
I think it will be a bit more readable if we have a dedicated header:

```suggestion
### Long response filtering

Output Filter input text box will be attached to your command response if response has more than 15 lines. Let's see an example for this situation.

```

WDYT?

Reply:
Done